docs/get-started/get-started.md

```diff
@@ -0,0 +1,206 @@
+
+# Get Started with Gravatar-SDK-Android
+
+## Setup
+
+You'll need an API key to use the SDK's full capabilities. You can use some features without an API Key, but you'll receive limited information, so we highly encourage you to create one in the [developers' portal](https://gravatar.com/developers/). 
+
+You can create the API Key as follows:
+
+1. Login in to your Gravatar account (or the Gravatar account you want to use to generate the key)
+2. Navigate to the [developers portal](https://gravatar.com/developers/)
+3. Tap on [`Create new Application`](https://gravatar.com/developers/new-application)
+4. Fill the required data and follow the flow
+
+<img src="screenshot-developers-portal.png" alt="Creating an API Key in the developers portal" width="600"/>
+
+5. You'll get your Gravatar API Key. Save it in a safe place!
+
+## Installation
+
+First step it to add the maven repositories to the file where you manage your the dependency resolution and the right dependencies to the `build.gradle` file:
+
+### Add the Gravatar dependencies to your project
+
+```groovy
+repositories {
+    maven {
+        url "https://a8c-libs.s3.amazonaws.com/android"
+    }
+    // Jitpack is used to fetch the uCrop library. Required only for gravatar-ui module.
+    maven {
+        url "https://jitpack.io"
+        content {
+            includeModule("com.github.yalantis", "ucrop")
+        }
+    }
+}
+```
+
+```groovy
+dependencies {
+    implementation ("com.gravatar:gravatar:<version>")
+    implementation ("com.gravatar:gravatar-ui:<version>")
+}
+```
+
+### Store the Gravatar API key in your app
+
+There are many ways to store the Gravatar API key in your app. The best way to do this depends on your app's architecture and requirements and how you're already storing other sensitive information. Make sure to avoid hardcoding the API key in your app's code and make sure to avoid storing it in a public repository.
+
+One way to API key in your app by adding it to the `local.properties` file:
+
+```properties
+gravatar.api.key = REPLACE_ME
+```
+
+Then update your gradle file to read the API key from the `local.properties` file and put it in the generated `BuildConfig` class:
+
+```groovy
+android {
+    buildFeatures.buildConfig = true
+
+    val properties = Properties()
+    properties.load(FileInputStream(project.rootProject.file("local.properties")))
+    buildConfigField("String", "GRAVATAR_API_KEY", "\"${properties["gravatar.api.key"]}\"")
+}
+```
+
+Then you can access the API key in your app's code like this:
+
+```kotlin
+Gravatar.initialize(BuildConfig.GRAVATAR_API_KEY)
+```
+
+## Usage
```

Comment:
Could we add a section on how to use the API methods to fetch a profile without displaying it in a UI component?

Reply:
Done! Great suggestion!